plugins/modules/digital_ocean_kubernetes_info.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2020, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: digital_ocean_kubernetes_info
+short_description: Returns information about an existing DigitalOcean Kubernetes cluster
+description:
+  - Returns information about an existing DigitalOcean Kubernetes cluster.
+version_added: 1.2.0
+author: Mark Mercado (@mamercad)
+options:
+  oauth_token:
+    description:
+      - DigitalOcean OAuth token; can be specified in C(DO_API_KEY), C(DO_API_TOKEN), or C(DO_OAUTH_TOKEN) environment variables
+    type: str
+    aliases: ['API_TOKEN']
+    required: yes
+  name:
+    description:
+      - A human-readable name for a Kubernetes cluster.
+    type: str
+    required: yes
+  return_kubeconfig:
+    description:
+      - Controls whether or not to return the C(kubeconfig).
+    type: bool
+    required: no
+    default: False
+requirements:
+  - python >= 2.6
```

Comment:
```suggestion
```
Ansible 2.9 and later do not support Python before 2.6 anyway, so it's not really necessary to mention that.